tox.ini

```diff
@@ -5,6 +5,7 @@ envlist = py38, black, flake8
 extras =
     test
 commands =
+    pip install git+https://github.com/asdf-format/asdf-schema-helpers
```

Comment:
Another option is to list this under `deps` in tox.ini:

```
deps =
    git+https://github.com/asdf-format/asdf-schema-helpers
```

Then, add `asdf-schema-helpers` to the test extras in setup.cfg.  Tox seems to install the `deps` first, which allows the setup.cfg install to succeed.

Reply:
I don't think `setup.cfg` can have pointers to git repos listed.  Or perhaps they have to be of the form:
```
"asdf-schema-helpers @ git+https://github.com/asdf-format/asdf-schema-helpers"
```

It would be nice to have dependencies listed in one location, not spread around 2 or more files.